pkg/lib/saml/samlprotocol/const.go

```diff
@@ -16,6 +16,11 @@ var SSOSupportedBindings []SAMLBinding = []SAMLBinding{
 	SAMLBindingHTTPPost,
 }
 
+var SLOSupportedBindings []SAMLBinding = []SAMLBinding{
+	SAMLBindingHTTPRedirect,
+	SAMLBindingHTTPPost,
+}
```

Comment:
do we actually going to both? I am asking this because in ACS, we support HTTP-Post only.

Reply:
I think yes, as logout response is much shorter so I guess using redirect binding is possible.

Reply:
Sorry this is not related to my previous comment.
This supported bindings is about what binding we accept in our slo service to receive a logout request. And yes redirect and post is accepted.
Like in SSO service, we also accept both bindings to receive a AuthnRequest.